helpers/artefact_helper.rb

```
@@ -0,0 +1,101 @@
+
+        model = model_from_type(type)
+
+        resource =
+          if model == 'property'
+            ontology.property(uri, latest_submission)
+          elsif model
+            model.find(uri).in(latest_submission).include(model.goo_attrs_to_load(includes_param)).first
+          end
+
+        reply resource
+      end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for resolve_resource_by_uri is too high. [<11, 27, 8> 30.23/23]

(Metrics/AbcSize)